docs/quick_start.rst

```diff
@@ -1,4 +1,131 @@
 How to write your first check
 ==============================
 
-# todo
\ No newline at end of file
+Basic idea of Contessa is to help you with defining data quality rules
```

Comment:
I would get rid of "you"s, i.e. `Basic idea of Contessa is to help with... ` then `Let's create a script ` -> `Create a script... `